sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -686,6 +694,10 @@ def receive_messages(
         self._check_live()
         if max_wait_time is not None and max_wait_time <= 0:
             raise ValueError("The max_wait_time must be greater than 0.")
+        if max_wait_time is not None and self._session_id is not None:
```

Comment:
can only specify it to session id being set not next_available because _open() chooses a session id to send to

Reply:
add to docstring